src/simulation_strategies/FixedLossSimulationStrategy.py

```diff
@@ -34,7 +35,7 @@ def __prepare_initial_state(self, input_state: ndarray) -> ndarray:
             :return:
         """
         initial_number_of_photons = sum(input_state)
-        prepared_state = input_state[:]
+        prepared_state = input_state
```

Comment:
Selector is not needed as whole matrix is taken.

Reply:
That was correct, but I got the state preparation wrong. This fix no longer applies, as you will see after the new push. :P